activecleaner.gemspec

```diff
@@ -16,20 +15,22 @@ Gem::Specification.new do |s|
 
   s.license = "MIT"
 
+  s.metadata["rubygems_mfa_required"] = "true"
+
   s.files = %w[README.md LICENSE]
   s.files += Dir.glob("lib/**/*")
   s.require_paths = ["lib"]
   s.test_files = Dir.glob("spec/**/*")
 
-  s.required_ruby_version = ">= 2.3.0"
+  s.required_ruby_version = ">= 2.5.0"
```

Comment:
It is sad we have to remove 2.3/2.4 support just for rubocop but it looks like we are in the future already and people really don't want to support EOL versions